cinema/serializers.py

```diff
@@ -1,6 +1,39 @@
 from rest_framework import serializers
+from cinema.models import Movie, Genre, Actor, CinemaHall
 
-from cinema.models import Movie
+
+class GenreSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    name = serializers.CharField(max_length=255)
+
+    def create(self, validated_data):
+        return Genre.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):
+        instance.name = validated_data.get("name", instance.name)
+        instance.save()
+        return instance
+
+
+class ActorSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` for `ActorSerializer` to simplify the code and automatically handle model fields.